arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -0,0 +1,1070 @@
+            "name": "SeverityTypes",
+            "modelAsString": false
+          }
+        },
+        "details": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the details of the error message."
+        },
+        "component": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the component that failed."
+        },
+        "errorId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the specific identifier for the type of error encountered in the job."
+        },
+        "helpLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link to MSDN or Azure help for this type of error, if any."
+        },
+        "internalDiagnostics": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the internal diagnostic stack trace if the user requesting the job error details has sufficient permissions it will be retrieved, otherwise it will be empty."
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the user friendly error message for the failure."
+        },
+        "resolution": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the recommended resolution for the failure, if any."
+        },
+        "source": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the ultimate source of the failure (usually either SYSTEM or USER)."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the error message description"
+        }
+      },
+      "description": "The Data Lake Analytics job error details."
+    },
+    "JobInformation": {
+      "properties": {
+        "jobId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "the job's unique identifier (a GUID)."
+        },
+        "name": {
+          "type": "string",
+          "description": "the friendly name of the job."
+        },
+        "type": {
+          "type": "string",
+          "description": "the job type of the current job (Hive or USql).",
+          "enum": [
+            "USql",
+            "Hive"
+          ],
+          "x-ms-enum": {
+            "name": "JobType",
+            "modelAsString": false
+          }
+        },
+        "submitter": {
+          "type": "string",
+          "description": "the user or account that submitted the job."
+        },
+        "errorMessage": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/JobErrorDetails"
+          },
+          "description": "the error message details for the job, if the job failed."
+        },
+        "degreeOfParallelism": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
Yes I believe the default the server chooses is 1. Also I have added the minimum, the max can't be set here since it is based on a different account level setting, which is configurable by the account admin.